src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -271,20 +683,57 @@
 		"sourcePath" : "log.sagebase.org/fileDownloadRecords/",
 		"tableDescriptor": {
 			"name": "filedownloadrecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"user_id": "bigint",
-				"project_id": "bigint",
-				"file_handle_id": "bigint",
-				"downloaded_file_handle_id": "bigint",
-				"association_object_id": "bigint",
-				"association_object_type": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contain records of all the downloads of the Synapse, e.g, file, zip/package, attachments. The events are recorded only after the pre-signed url for requested download entity is generated.",
+			"columns":[{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the file download event is pushed to the queue for recording, after generating the Pre-signed url."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The id of the user who downloaded the file."
+			},
+			{
+				"name": "project_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the project where the downloaded entity resides. Applicable only for FileEntity and TableEntity."
+			},
+			{
+				"name": "file_handle_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the file handle."
 			},
+			{
+				"name": "downloaded_file_handle_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the file handle of a zip file when the download is requested as zip/package otherwise file_handle_id is downloaded_file_handle_id."
```

Comment:
The unique identifier of the zip file handle containing the downloaded file when the download is requested as zip/package, otherwise the id of the file handle itself.